platformcomponents/desktop/global.json

```diff
@@ -0,0 +1,9 @@
+{
+  "global": {
+    "comment": "For windows global outline for all focused components",
+    "figma": "https://www.figma.com/file/mZjOAoZIKLYFbOoG0h0r1A/Focus-States?node-id=812%3A53922",
+    "focus": {
+      "outline": "@theme-control-active-normal"
+    }
+  }
+}
```

Comment:
Original source of this token:

```json
{
  "global-focus": {
    "figma": "https://www.figma.com/file/mZjOAoZIKLYFbOoG0h0r1A/Focus-States?node-id=812%3A53922",
    "#normal": {
      "outline": "@theme-outline-theme-normal"
    }
  }
}
```

Reply:
`theme-outline-theme-normal` was used based on the spec above but not completely happy with the colors generated(generates 1 single color but it should be themed) for `theme-outline-theme-normal` from this repo.